.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,11 +4,13 @@ Please fill in the following checklist, removing items that do not apply.
 See also https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md
 -->
 
-- [ ] The page (if new), does not already exist in the repository.
-- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
-- [ ] The page has 8 or fewer examples.
-- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
-- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
-- [ ] The page description includes a link to documentation or a homepage (if applicable).
+# Command info
+
+- **Version of the command being documented:**  
 
-**Version of the command being documented (if known):**
+- [ ] New command pages or translations are added.
```

Comment:
Also a query on this one.
PRs don't have to add new pages or translations, we sometimes have typos, poor formatting, localization issues, etc.

Reply:
I agree😁

Reply:
Then maybe remove this one, or change it back to what we had before? ^-^'

People that want to update the README, update an existing page, or improve our documentation/templates don't have to add new pages.

Reply:
Then I think I have to remove `- [ ] All command pages have 8 or fewer examples.` because README for instance isn't a tl;dr page with just 8 examples.

Reply:
I like this idea. PR goal can be specified in it's title and there is no need to explicitly write about addition new pages or update of existing pages.